Example_StaticLibrary/PointziDemo/Supporting Files/Info.plist

```diff
@@ -32,7 +32,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>1</string>
+	<string>IOS1121_GitVersion-e1d7a67b-Wed Sep 20 14:34:28 CST 2017</string>
```

Comment:
this string is fixed ?

Reply:
No, it's updated with each build. 
When CI runs, this repository will dirty with new pod binary, version etc.